recipes/beyla-golden-signals/gce/README.md

```diff
@@ -0,0 +1,90 @@
+# eBPF Golden Signals with Beyla on GCE
+
+This recipe demonstrates how to configure the Google Cloud Ops Agent (installed on your Google Compute Engine instance) to produce golden signal metrics from [Beyla](https://github.com/grafana/beyla) http
+metrics and send those metrics to [Google Managed Service for
+Prometheus](https://cloud.google.com/stackdriver/docs/managed-prometheus).
+
+In this recipe, Beyla is configured to generate http metrics from an application running on a GCE instance without introducing any code changes. The sample application used for this recipe is a simple Java server-client application located in the [sample-apps/java](../../../sample-apps/java/) directory of this repository.
+
+This recipe is based on applying an [Google Cloud Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent) configuration that enables the agent to receive and export telemetry generated by Beyla to [Google Managed Service for Prometheus](https://cloud.google.com/stackdriver/docs/managed-prometheus).
+
+Beyla allows [two modes of exporting data](https://grafana.com/docs/beyla/latest/configure/export-modes/#beyla-export-modes). This sample relies on the *Direct Mode* which allows Beyla to push OTLP data to a specified endpoint.
+
+# Prerequisites
+
+* Cloud Monitoring API enabled in your GCP project
+* The `roles/monitoring.metricWriter`
+  [IAM permissions](https://cloud.google.com/trace/docs/iam#roles) for your cluster's service
+  account (or Workload Identity setup as shown below).
+* A GCE instance running Debian Linux and [Google Cloud Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent) installed.
+    * For various installation methods see the [installation instructions](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/install-index).
+    * If you're creating the GCE instance from Google Cloud Console, you can simply check the `Install Ops Agent for Monitoring and Logging` checkbox during VM creation.
+* Ability to SSH into the created GCE instance.
+* The files in this recipe, locally on the created VM.
+  * You can `git clone` this repository directly on the VM and `cd` into this recipe's folder. *You may need to install `git` on the VM.*
+
+## Running
+
+From the location of this recipe *(beyla-golden-sigals/gce)*, execute the following:
+
+### Configure the VM instance
+
+#### Install and configure Beyla on the VM:
+
+```sh
+# Make sure to run this without a dot slash (./) since it needs to source variables.
+# Running using ./ will run the script it in a subshell. 
+. configure-beyla.sh
+```
+
+#### Check if Google Cloud Ops Agent is still running (Optional):
+
+```sh
+# Logging Agent and Metrics Agent should be running
+sudo systemctl status "google-cloud-ops-agent*"
+```
+
+#### Start Beyla process
+
+```sh
+# Start beyla in the background
```

Comment:
Maybe add a big warning that we are running beyla as root?

Reply:
Sounds good